ui/pages/add-token/add-token.component.js

```diff
@@ -211,16 +220,20 @@ class AddToken extends Component {
   }
 
   handleCustomDecimalsChange(value) {
-    const customDecimals = value.trim();
-    const validDecimals =
-      customDecimals !== null &&
-      customDecimals !== '' &&
-      customDecimals >= MIN_DECIMAL_VALUE &&
-      customDecimals <= MAX_DECIMAL_VALUE;
-    let customDecimalsError = null;
+    const inValidDecimals = !(
+      value >= MIN_DECIMAL_VALUE && value <= MAX_DECIMAL_VALUE
+    );
 
-    if (!validDecimals) {
-      customDecimalsError = this.context.t('decimalsMustZerotoTen');
+    let customDecimals;
+    let customDecimalsError = null;
+    if (value) {
+      customDecimals = Number(value.trim());
```

Comment:
I think we should define `customDecimals` at the beginning of this function so we can fail early if needed and use it in the validation checks

Reply:
I have moved the validity check to inside the if condition, so now the function processing starts by checking for the existence of value. Our lint rules do not allow the negated check so we can't check for `if(!value)`